apps/api/v2/src/ee/bookings/controllers/bookings.controller.e2e-spec.ts

```diff
@@ -0,0 +1,205 @@
+import { UsersModule } from "@/modules/users/users.module";
+import { INestApplication } from "@nestjs/common";
+import { NestExpressApplication } from "@nestjs/platform-express";
+import { Test } from "@nestjs/testing";
+import { User } from "@prisma/client";
+import * as request from "supertest";
+import { UserRepositoryFixture } from "test/fixtures/repository/users.repository.fixture";
+import { withAccessTokenAuth } from "test/utils/withAccessTokenAuth";
+
+import { SUCCESS_STATUS } from "@calcom/platform-constants";
+import {
+  getAllUserBookings,
+  handleNewBooking,
+  getBookingInfo,
+  handleNewRecurringBooking,
+  handleInstantMeeting,
+} from "@calcom/platform-libraries";
+import { ApiSuccessResponse, ApiResponse } from "@calcom/platform-types";
+
+describe("Bookings Endpoints", () => {
+  describe("User Authentication", () => {
+    let app: INestApplication;
+
+    let userRepositoryFixture: UserRepositoryFixture;
+
+    const userEmail = "bookings-controller-e2e@api.com";
+    let user: User;
+
+    let createdBooking: Awaited<ReturnType<typeof handleNewBooking>>;
+
+    beforeAll(async () => {
+      const moduleRef = await withAccessTokenAuth(
+        userEmail,
+        Test.createTestingModule({
+          imports: [AppModule, PrismaModule, AvailabilitiesModule, UsersModule],
+          providers: [SchedulesRepository, SchedulesService],
+        })
+      ).compile();
+
+      userRepositoryFixture = new UserRepositoryFixture(moduleRef);
+
+      user = await userRepositoryFixture.create({
+        email: userEmail,
+      });
+
+      app = moduleRef.createNestApplication();
+      bootstrap(app as NestExpressApplication);
+
+      await app.init();
+    });
+
+    it("should be defined", () => {
+      expect(userRepositoryFixture).toBeDefined();
+      expect(user).toBeDefined();
+    });
+
+    it("should create a booking", async () => {
+      const bookingStart = "2023-05-25T09:30:00.000Z";
+      const bookingEnd = "2023-05-25T10:30:00.000Z";
+      const bookingEventTypeId = 7;
+      const bookingTimeZone = "Europe/Londom";
+      const bookingLanguage = "en";
+      const bookingHashedLink = "";
+      const bookingMetadata = {};
+
+      const body = {
+        start: bookingStart,
+        end: bookingEnd,
+        eventTypeId: bookingEventTypeId,
+        timeZone: bookingTimeZone,
+        language: bookingLanguage,
+        metadata: bookingMetadata,
+        hashedLink: bookingHashedLink,
+      };
+
+      return request(app.getHttpServer())
+        .post("/api/v2/ee/bookings")
+        .send(body)
+        .expect(201)
+        .then(async (response) => {
+          const responseBody: ApiSuccessResponse<Awaited<ReturnType<typeof handleNewBooking>>> =
+            response.body;
+          expect(responseBody.status).toEqual(SUCCESS_STATUS);
+          expect(responseBody.data).toBeDefined();
+          expect(responseBody.data.user.email).toBeDefined();
```

Comment:
Is this the e-mail of "bookings-controller-e2e@api.com" ? if so, we can make test more specific by checking that user.email equals "bookings-controller-e2e@api.com"